lib/evmone/analysis.cpp

```diff
@@ -54,38 +54,20 @@ code_analysis analyze(
     block_info* block = nullptr;
 
     int block_stack_change = 0;
-    int instr_index = 0;
+
+    // Create new block.
+    block = &analysis.blocks.emplace_back();
+    block_stack_change = 0;
+    analysis.instrs.emplace_back(fns[OPX_BEGINBLOCK]).arg.p.number =
```

Comment:
This was clearer when it was on two separate lines.

Reply:
Done.